functions-methods.js

```diff
@@ -20,7 +26,18 @@
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
 
+function typeOfEmail(email) {
+    if (email.includes("@novi-education.nl")) {
+        return email = "Student";
+    } else if (email.includes("@novi.nl")) {
+        return email = "Medewerker";
+    } else if (email.includes("@outlook.com") || (email.includes("@gmail.com"))) {
```

Comment:
Je zou hier ook gewoon een else van kunnen maken. In principe geef je niet meer om het domein als de eerste twee if's niet true opleveren
```suggestion
    } else {
```